### ISSUE TITLE
Weird Initialization of IMU + Stereo Camera

### ISSUE THREAD

**Issue by djramakrishna** [HKUST-Aerial-Robotics/VINS-Fusion#198]
Hi all ! I'm using VINS-Fusion for my custom bags with stereo camera + IMU. The weird thing i see is, the animation of the bot in the Rviz, moves pretty fast in the beginning, without even my camera frame moving. But once, my robot starts, the path obtained is as expected. It's just that there's a pretty long initial path that is generated without the robot even moving. I'm not sure what's happening and any thoughts about this appreciated. Thank you !

**Comment by rreddy78** [comment]
Yes. This is some behaviour even I have observed. I am testing VINS-Fusion with a CARLA simulator output.

**Comment by SharkPeppero** [comment]
> Hi all ! I'm using VINS-Fusion for my custom bags with stereo camera + IMU. The weird thing i see is, the animation of the bot in the Rviz, moves pretty fast in the beginning, without even my camera frame moving. But once, my robot starts, the path obtained is as expected. It's just that there's a pretty long initial path that is generated without the robot even moving. I'm not sure what's happening and any thoughts about this appreciated. Thank you !

Hello, I seem to be encountering the same problem as you.

[Problem Description]: In binocular IMU mode, the system experiences state 
jumps when transitioning from a stationary to a moving phase.

[My Thoughts]: Is it because too many keyframes are inserted during the initialization phase, leading to instability in the factor graph when motion starts?

[How to Solve]: Currently, I haven't thought of a good solution. I tried adding ZUPT (Zero-velocity Update); when stationary, I added observations with a speed of 0. During motion, the ZUPT weight decreases, but the effect is not very good.

I wonder if you have solved this problem?

Looking forward to your reply.
